src/Data/Models/DicomSeriesDTO.php

```diff
@@ -48,21 +48,21 @@ class DicomSeriesDTO
      * @param ?int    $id             The TarchiveSeriesID
      * @param ?string $description    The SeriesDescription
      * @param ?int    $number         The SeriesNumber
-     * @param ?string $echotime       The EchoTime
-     * @param ?string $repetitiontime The RepetitionTime
-     * @param ?string $inversiontime  The InversionTime
-     * @param ?string $slicethickness The SliceThickness
+     * @param ?float $echotime       The EchoTime
```

Comment:
shouldn't `float` be `int` in this file?

Reply:
Echo time is a float in our database, so I think it should be a float.

Reply:
But isn't RepetitionTime in the DB also a float for example and it is set as string in the API?

Reply:
Oh yep, but I reverted the change and made it back to a string !

The most up-to-date comment of mine is this one. It is true I did not update each point individually (probably should have, sorry !)
https://github.com/aces/Loris/pull/9154#issuecomment-2356840330